bip-0321.mediawiki

```diff
@@ -0,0 +1,222 @@
+<pre>
+  BIP: XXXX
+  Layer: Applications
+  Title: URI Scheme
+  Author: Matt Corallo <bip21@bluematt.me>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-XXXX
+  Status: Draft
+  Type: Standards Track
+  Created: 2024-07-13
```

Comment:
The created date refers to the number assignment. (Perhaps we should also change that in BIP 3, while we are at it.)

```suggestion
  Created: 2024-11-15
```